keps/sig-contributor-experience/20190121-artifact-management.md

```diff
@@ -0,0 +1,124 @@
+---
+kep-number: 34
+title: Kubernetes Community Artifact Serving
+authors:
+  - "@brendandburns"
+owning-sig: sig-contributor-experience
```

Comment:
strong disagree that contribex should own this, IMO this is sig-release, under their release-engineering subproject

Reply:
I would be more comfortable with the merge and iterate approach if we got this right

Reply:
+100.  I think that comment was also made in previous versions too.

Reply:
+1 about sig-release.

cc @cblecker